runtime/plugins/linter/help/linter.md

```diff
@@ -24,6 +24,14 @@ following filetypes and linters:
 * **swift**: swiftc (MacOS and Linux only)
 * **yaml**: yamllint
 
+You can easily enable linting in your `settings.json` file:
```

Comment:
I would also mention the easy way to do this which is the `set linter on|off` command.